app/main.py

```diff
@@ -1 +1,50 @@
-#  write your code here
+class Animal:
+
+    def __init__(
+            self, name: str, appetite: int,
+            is_hungry: bool = True) -> None:
+
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if not self.is_hungry:
+            return 0
+
+        print(f"Eating {self.appetite} food points...")
+        self.is_hungry = not self.is_hungry
```

Comment:
The line `self.is_hungry = not self.is_hungry` toggles the `is_hungry` state. This is correct if the intention is to make the animal not hungry after feeding. Ensure this behavior aligns with your requirements.